footer.css

```diff
@@ -85,7 +85,38 @@ footer{
     margin-left:15px;
 }
 
+@media screen and (min-width: 834px) {
+    footer {
+      width: 834px;
+      height: 625px;
+    }
+    #logoAdressContactSection{
```

Comment:
Are the ID's used for JavaScript purposes? Otherwise, change to the more re-usable class syntax :)

Reply:
thanks, will change to classes instead!